docs/src/architecture.md

```diff
@@ -1,59 +1,52 @@
 # Architecture
-The Polygon Miden Architecture decribes the concepts of how the participants of the network can interact with each other. The Architecture reflects the design goal: We aim to build an Ethereum scaling solution that extends its feature set. Rollups allow the creation of new design spaces while retaining the collateral security of Ethereum. This is where to innovate, whereas the base layer should provide stability and evolve slowly.
+The Polygon Miden Architecture decribes the concepts of how the participants of the network can interact with each other. The architecture reflects the design goal: We aim to build an Ethereum scaling solution that extends its feature set. We want to enable developers to build faster, safer and more private dApps. Rollups allow the creation of new design spaces while retaining the collateral security of Ethereum. This is where to innovate, whereas the base layer should provide stability and evolve slowly.
 
-We believe that to attract billions of users to Ethereum, we need to extend it on three dimensions: scalability, safety, and privacy
-
-## Scalability
-
-## Safety
-
-## Privacy
-
-Users can interact on the network executing transactions. Transactions are always executed against single accounts and produce or consume at least one note. The state of all accounts and notes is compressed into the L2 state root which is being published and accompanied by a ZK proof and additional data to Ethereum.
+We believe that to attract billions of users to Ethereum, we need to extend it on three dimensions: scalability, safety, and privacy. 
 
 The actor model is our inspiration for achieving concurrent, local state changes on a distributed system like a blockchain. In the actor model, actors play the role of little state machines, meaning each actor is responsible for their own state. Actors have inboxes to send and receive messages to communicate with other actors. Messages can be read asynchronously.
 
-The concepts which constitutes the Miden architecture are 
+## Concepts in Miden
+Users can interact on the network executing transactions. Transactions are always executed against single accounts and produce or consume at least one note. This is similar to the [Actor Model](https://en.wikipedia.org/wiki/Actor_model). The state of all accounts and notes is committed to via the L2 state root which is being published and accompanied by a ZK proof and additional data to Ethereum. This is similar to any other zkRollup. 
```

Comment:
Let me think about it. To me, transactions are being "executed", and this can happen client-side or network-side. Regardless of how the transaction is being executed, it is the only way to interact on the network (or to write on the database). Would you agree?

Reply:
Yes I would agree that transactions are the single entry point for the network.